Artesian/Artesian.SDK/Service/Query/QueryParamaters/QueryParamaters.cs

```diff
@@ -62,6 +65,10 @@ public QueryParamaters()
         /// filterId
         /// </summary>
         public int? FilterId { get; set; }
+        /// <summary>
+        /// Filler Kind
+        /// </summary>
+        public FillerKind FillerKind { get; set; } = new FillerKind();
```

Comment:
FillerKind isn't an Enum? better to write FillerKind.Default then using `new`, as is clearer to read